lms/djangoapps/certificates/admin.py

```diff
@@ -2,7 +2,8 @@
 django admin pages for certificates models
 """
 from django.contrib import admin
-from certificates.models import CertificateGenerationConfiguration
+from certificates.models import CertificateGenerationConfiguration, CertificateHtmlViewConfiguration
 
 
 admin.site.register(CertificateGenerationConfiguration)
+admin.site.register(CertificateHtmlViewConfiguration)
```

Comment:
I found a couple examples of this in the platform -- is it really as simple as it appears in say, https://github.com/edx/edx-platform/blob/master/cms/djangoapps/contentstore/admin.py -- just import the class and include in the register() instruction?  Seems like there should be more too it than that...